### PR TITLE
Qt: Call Render::SetFullscreen

### PR DESCRIPTION
**Pull request by spycrab**

### CODE REVIEW

File: Source/Core/DolphinQt2/Host.cpp [hunk 1]
```diff
@@ -53,6 +54,10 @@ bool Host::GetRenderFullscreen()
 void Host::SetRenderFullscreen(bool fullscreen)
 {
   m_render_fullscreen = fullscreen;
+
+  if (!g_renderer || g_renderer->IsFullscreen() == fullscreen ||
```
Comment (stenzek) [inline Source/Core/DolphinQt2/Host.cpp]: This logic doesn't seem right to me. If there is no renderer, or the fullscreen state is already matching, call SetFullscreen? Should it be `if (g_ActiveConfig.ExclusiveFullscreenEnabled() && g_renderer && g_renderer->IsFullscreen() != fullscreen)`?

Comment (spycrab) [reply Source/Core/DolphinQt2/Host.cpp]: Yep, fucked up a bit there.

**Comment by stenzek** [comment]
For this to restore exclusive full-screen after alt-tabbing, you should run the same code when the render widget is activated/receives focus as well. Otherwise, it only works until you switch windows.

**Comment by stenzek** [comment]
Not 100% sure if this is true "exclusive" mode, but I'm seeing SetFullscreenState return success, so LGTM!

**Pull request merged**